### PR TITLE
new events on top #3125

### PR DESCRIPTION
**Pull request by Himanshi-Khandelwal**
latest events on top.

### CODE REVIEW

File: app/helpers/data_getter.py [hunk 1]
```diff
@@ -98,7 +98,7 @@ def get_invite_by_user_id(user_id):
     @staticmethod
     def get_all_events():
         """Method return all events"""
-        return Event.query.order_by(desc(Event.id)).filter_by(in_trash=False).all()
+        return Event.query.order_by(Event.timezone).filter_by(in_trash=False).all()
```
Comment (SaptakS) [inline app/helpers/data_getter.py]: How does ordering by Event.timezone make new events to come on top?? Can you please explain?

File: app/helpers/data_getter.py [hunk 2]
```diff
@@ -98,7 +98,7 @@ def get_invite_by_user_id(user_id):
     @staticmethod
     def get_all_events():
         """Method return all events"""
-        return Event.query.order_by(desc(Event.id)).filter_by(in_trash=False).all()
+        return Event.query.order_by(Event.created_at).filter_by(in_trash=False).all()
```
Comment (SaptakS) [inline app/helpers/data_getter.py]: Shouldn't this be in `desc(created_at)` ?? Please check your code after making changes before pushing it. Thanks...

Comment (Himanshi-Khandelwal) [inline app/helpers/data_getter.py]: sry , actually I have not created enough events in my database. That's y.

Comment (SaptakS) [inline app/helpers/data_getter.py]: Please do... Thanks...

File: app/helpers/data_getter.py [hunk 3]
```diff
@@ -98,8 +98,8 @@ def get_invite_by_user_id(user_id):
     @staticmethod
     def get_all_events():
         """Method return all events"""
-        return Event.query.order_by(desc(Event.id)).filter_by(in_trash=False).all()
-
+        return Event.query.order_by(desc(Event.created_at)) .filter_by(in_trash=False).all()
```
Comment (SaptakS) [inline app/helpers/data_getter.py]: Please remove the spaces. I doubt line 101 even works. Please check code before pushing.

**Pull request closed without merging**